services/terrain/src/terrain/services/admin.clj

```diff
@@ -40,11 +40,14 @@
   [url-to-scrub]
   (str (url/url url-to-scrub :path "/")))
 
+(defn get-with-timeout
+  [url]
+  (client/get url {:socket-timeout 1000 :conn-timeout 1000}))
```

Comment:
/shrug My gut says 5~10 seconds would be better, in case of network lag and not because the JEX is down?